client/src/backend/containers/records/export-targets/Edit.js

```diff
@@ -0,0 +1,112 @@
+import React, { PureComponent } from "react";
+import PropTypes from "prop-types";
+import connectAndFetch from "utils/connectAndFetch";
+import withConfirmation from "hoc/with-confirmation";
+import { entityStoreActions } from "actions";
+import { select } from "utils/entityUtils";
+import { exportTargetsAPI, requests } from "api";
+import lh from "helpers/linkHandler";
+import Navigation from "backend/components/navigation";
+import Form from "./Form";
+
+const { request, flush } = entityStoreActions;
+
+export class ExportTargetsEditContainer extends PureComponent {
+  static mapStateToProps = state => {
+    return {
+      exportTarget: select(requests.beExportTarget, state.entityStore)
+    };
+  };
+
+  static displayName = "ExportTargets.Edit";
+
+  static propTypes = {
+    match: PropTypes.object.isRequired,
+    confirm: PropTypes.func.isRequired,
+    dispatch: PropTypes.func.isRequired,
+    exportTarget: PropTypes.object,
+    history: PropTypes.object.isRequired
+  };
+
+  static defaultProps = {
+    exportTarget: {}
+  };
+
+  componentDidMount() {
+    this.fetchExportTarget(this.exportTargetID);
+  }
+
+  componentDidUpdate(prevProps) {
+    if (prevProps.match.params.id !== this.exportTargetID) {
+      this.fetchExportTarget(this.exportTargetID);
+    }
+  }
+
+  componentWillUnmount() {
+    this.props.dispatch(flush([requests.beExportTargetUpdate]));
```

Comment:
You're flushing a request that's not really generated in this component. You should be flushing requests. beExportTarget